### ISSUE TITLE
Wrong initialization of the temporary directory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#587]
```
Running Refine in some environments, file import throws an error:

Error uploading data .import-temp/1341219847990/raw-data/foo.csv (No such file 
or directory)

Debugging it, Sebastian Schaffert discovered that the source of the issue would 
be a wrong usage of a local variable with the same name of the class field. See 
the path here:

https://bitbucket.org/wikier/google-refine/changeset/f3dbdb16a320#chg-main/src/c
om/google/refine/RefineServlet.java
```

Original issue reported on code.google.com by `sergio.fernandez@fundacionctic.org` on 2 Jul 2012 at 10:28

**Comment by GoogleCodeExporter** [comment]
```
typo at the title: s/directoty/directory
```

Original comment by `sergio.fernandez@fundacionctic.org` on 2 Jul 2012 at 1:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 18 Sep 2012 at 8:25
- Changed state: **Accepted**
- Added labels: **Milestone-2.6**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 13 Oct 2012 at 3:50
- Changed title: **Wrong initialization of the temporary directory**
- Changed state: **Started**
- Added labels: **Patch-NeedsReview**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r2583.
```

Original comment by `tfmorris` on 13 Oct 2012 at 4:00
- Changed state: **Fixed**
- Added labels: **Patch-Applied**
- Removed labels: **Patch-NeedsReview**

**Comment by GoogleCodeExporter** [comment]
```
How can i fix this on a currently installed version of Google Refine? New 
downloads don't seem to address the issue.
```

Original comment by `susanemc...@gmail.com` on 7 Nov 2013 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
This issue tracker is no longer being used.  The current one is 
https://github.com/OpenRefine/OpenRefine/issues/587

Are you saying that the bug doesn't appear to be fixed in OpenRefine 2.6 beta1? 
 If so please update new bug tracker with your environment, software versions, 
the error message, etc.
```

Original comment by `tfmorris` on 7 Nov 2013 at 10:15
